site2/docs/schema-manage.md

```diff
@@ -50,32 +58,47 @@ For a consumer, the `AutoUpdate` happens in the following cases:
 
 * If a **consumer connects to a topic without a schema** (which means the consumer receiving raw bytes), the consumer can connect to the topic successfully without doing any compatibility check.
 
-* If a **consumer connects to a topic with a schema**:
-
-  * If the **topic is idle** (no producers, no entries, no other consumers and no registered schemas), the broker registers a schema for the topic automatically.
+* If a **consumer connects to a topic with a schema**.
 
-  * If the **topic is not idle**, the broker verifies if the schema provided by the consumer is compatible with the registered schema of the topic. 
+    * If a topic does not have all of them (a schema/data/a local consumer and a local producer):
     
-    * If the **schema passes the compatibility check**, the consumer can connect to the topic and receive messages. 
+        * If `isAllowAutoUpdateSchema` sets to **true**, then the consumer registers a schema and it is connected to a broker.
+        
+        * If `isAllowAutoUpdateSchema` sets to **false**, then the consumer is rejected to connect to a broker.
+        
+    * If a topic has one of them (a schema/data/a local consumer and a local producer), then the schema compatibility check is performed.
     
-    * If the **schema does not pass the compatibility check**, the consumer is rejected and disconnected.
-
-![AutoUpdate Producer](assets/schema-autoupdate-consumer.png)
+        * If the schema passes the compatibility check, then the consumer is connected to the broker.
+        
+        * If the schema does not pass the compatibility check, then the consumer is rejected to connect to the broker.
+        
+![AutoUpdate Consumer](assets/schema-autoupdate-consumer.png)
+        
 
 ### Manage AutoUpdate strategy
 
 You can use the `pulsar-admin` command to manage the `AutoUpdate` strategy as below:
 
+* [Enable AutoUpdate](#enable-autoupdate)
+
 * [Disable AutoUpdate](#disable-autoupdate)
```

Comment:
As we discussed just now, please correct the commands for `enable` and `disable`